07.ls_object/display.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class Display
+  FILE_TYPE = { 'file' => '-', 'directory' => 'd', 'link' => 'l' }.freeze
+  PERMISSION = { '0' => '---', '1' => '--x', '2' => '-w-', '3' => '-wx', '4' => 'r--', '5' => 'r-x', '6' => 'rw-', '7' => 'rwx' }.freeze
+  COLUMN = 3
+
+  def initialize(file_info_list, options)
+    @file_info_list = file_info_list
+    @options = options
+  end
+
+  def show_files
```

Comment:
newした時点でlongかnormalを決めてしまうのではなく、表示のタイミングで切り替えられるようにした方がクラスとしては便利かも知れません。

```suggestion
  def show_files(long: false)
```

Reply:
ありがとうございます🙏